app/docker/views/dashboard/dashboard.html

```diff
@@ -84,7 +84,7 @@
             <i class="fa fa-th-list"></i>
           </div>
           <div class="title">{{ stackCount }}</div>
-          <div class="comment">Stacks</div>
+          <div class="comment">Stack<span ng-if="stackCount === 0 || stackCount > 1">s</span></div>
```

Comment:
I believe that we should use a clearer pattern using variable interpolation here:

```suggestion
          <div class="comment">{{ stackCount === 0 ? 'Stack' : 'Stacks' }}</div>
```

Reply:
I agree with you about using clearer patterns :+1:. The ternary operator's condition should probably be `stackCount === 1`

Reply:
Yup, I review it too quickly last night, sorry :-)